workflows/proteomics/openms-metaprosip/.dockstore.yml

```
@@ -0,0 +1,7 @@
+version: 0.1
```

This is the version of the dockstore yml file
```suggestion
version: 1.2
```